src/e2e/puppeteer/__tests__/toggleSidebar.ts

```diff
@@ -0,0 +1,29 @@
+import click from '../helpers/click'
+import getSelection from '../helpers/getSelection'
+import paste from '../helpers/paste'
+import waitForEditable from '../helpers/waitForEditable'
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+describe('all platforms', () => {
```

Comment:
```suggestion
describe('toggle sidebar', () => {
```

The `describe` title should more accurately describe the set of test cases.